docker-compose-dev.yml

```diff
@@ -59,9 +59,13 @@ services:
       - ./containers:/app
       - ./results:/results
       - ./logs:/logs
-    runtime: nvidia
-    devices:
-      - "/dev/nvidia0:/dev/nvidia0"
+    deploy:
```

Comment:
@alejoe91 can you please check if you have docker compose > 2 installed?

otherwise, what is the error you find?
This seems to be the most current way to give containers access to GPU through compose, the other method, as you mentioned, seems to be deprecated (although it was still running fine for me, weirdly enough)

Reply:
@alejoe91 another possible solution is this: https://docs.docker.com/config/containers/resource_constraints/#access-an-nvidia-gpu

can you please try and let me know if you needed to update your compose or if installing `nvidia-container-runtime` solved for you? We should then include these requirements in the description of the project